### ISSUE TITLE
Crash when opening parameters with 1.3.0

### ISSUE THREAD

**Issue by ipoupaille** [stefan-niedermann/nextcloud-deck#493]
When opening paramaters or about, the app crash:

```
App Version: 1.3.0
App Version Code: 1003000
App Flavor: fdroid

Files App Version Code: 30110190

---

OS Version: 3.10.84-perf-ge1ecd45(6)
OS API Level: 24
Device: kinzie
Manufacturer: motorola
Model (and Product): XT1580 (kinzie_reteu)

---

java.lang.RuntimeException: Unable to stop activity {it.niedermann.nextcloud.deck/it.niedermann.nextcloud.deck.ui.MainActivity}: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
	at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:3861)
	at android.app.ActivityThread.handleStopActivity(ActivityThread.java:3910)
	at android.app.ActivityThread.-wrap25(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1507)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6123)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
Caused by: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
	at androidx.fragment.app.FragmentManager.checkStateLoss(FragmentManager.java:1691)
	at androidx.fragment.app.FragmentManager.enqueueAction(FragmentManager.java:1731)
	at androidx.fragment.app.BackStackRecord.commitInternal(BackStackRecord.java:321)
	at androidx.fragment.app.BackStackRecord.commit(BackStackRecord.java:286)
	at it.niedermann.nextcloud.deck.ui.MainActivity.onStop(MainActivity.java:368)
	at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1311)
	at android.app.Activity.performStop(Activity.java:6847)
	at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:3858)
	... 9 more

```

**Comment by stefan-niedermann** [comment]
Should be fixed with `1.4.0` (already tagged), but i will double check this before closing this issue.

**Comment by stefan-niedermann** [comment]
Jup, will be fixed in `1.4.0`.

**Comment by stefan-niedermann** [comment]
Thank you for the report though :wink:
